Fix checks

mblue9 opened the pull request:
I've fixed the errors, warnings, and some of the notes.

mblue9 commented:
I expanded the description a little to fix the Note 
"The Description field in the DESCRIPTION is made up by less\nthan 3 sentences. Please consider expanding this field, and\nstructure it as a full paragraph"

see what you think I can edit

mblue9 commented:
wondering about join_transcripts function - the documentation in methods.R says it uses edgeR is that copy/paste carryover?

stemangiola commented:
> wondering about join_transcripts function - the documentation in methods.R says it uses edgeR is that copy/paste carryover?

You know me well! XD Yeah sometime, I forget to update things. That's what happened when you make two working repos in 4 days!

Thanks for noting all these things!

stemangiola commented:
As a note, this repository is a mirror of tisyseurat, so if you note something wrong here, if you happen to leave an issue on tidyseurat I will remember to fix that as soon as I can. But as you notice you have now access to all our repos, so feel at home!

stemangiola commented on R/plotly_methods.R:
```diff
@@ -92,7 +92,6 @@
 #' @return A plotly
 #' @export
 #' @examples
-#' \dontrun{
```
in my experience plotly does not play well with remote tests, I think it is fair enough to leave it out as norun.

Most servers do not have javascript libraries needed for interactive plotting

stemangiola replied:
the norun, is taken from plotly library itself

stemangiola replied:
Here you go :) the test fail. I will revert this edit
```
  > # plot_ly() tries to create a sensible plot based on the information you 
  > # give it. If you don't provide a trace type, plot_ly() will infer one.
  > plot_ly(economics, x = ~pop)
  Error in plot_ly(economics, x = ~pop) : object 'economics' not found
  Execution halted
```

mblue9 replied:
Ok fine, I think it's still failing because the Rd needs reverting too. Are you able to do that? Or want me to?

stemangiola replied:
> 
> 
> Ok fine, I think it's still failing because the Rd needs reverting too. Are you able to do that? Or want me to?

I'll do it no worries

stemangiola commented on R/utilities.R:
```diff
@@ -77,7 +79,6 @@ drop_class = function(var, name) {
 #' @return A tidySCE object
 #'
 #'
-#' @export
```
Of course these received complains, I forgot to take off the @export! :) Thanks

mblue9 commented:
1. Can you check the `join_transcripts` documentation? 
I wasn't sure what exactly `exclude_zeros` is doing is it excluding cells with zero values?
And what does `all` do? 

2. I removed dontrun from the plotly examples to remove the Note about is that ok?

3. I removed @export from these 2 internal functions as think don't need to export to user?
  ‘get_abundance_sc_long’ ‘get_abundance_sc_wide’

4. Should I remove @inheritParams from these two?
Topic 'as_tibble': no parameters to inherit with @inheritParams 
Topic 'extract': no parameters to inherit with @inheritParams 

5. Does this need a @name ?
tidy.SingleCellExperiment.Rd is missing name/title. Skipping 

6. Should we try fix the warning from these tests? 

```
test-dplyr.R:8: warning: arrange
You're computing too large a percentage of total singular values, use a standard svd instead.

test-dplyr.R:9: warning: arrange
You're computing too large a percentage of total singular values, use a standard svd instead.

test-tidyr.R:12: warning: nest_unnest
You're computing too large a percentage of total singular values, use a standard svd instead.

test-tidyr.R:13: warning: nest_unnest
You're computing too large a percentage of total singular values, use a standard svd instead.
```

stemangiola commented:
Thanks @mblue9 

.1 great
.2 reverted test fail (I explained why in the review)
.3 great
.4 great
.5 not sure I think it needs name
.6 no worries those are scran scater warnings, nothing to do with us

stemangiola commented:
@mblue9 all checks have passed! Can I merge?

mblue9 commented:
Yep go for it.
